docbook/Admin_Guide/en-US/config/api.xml

```diff
@@ -2,11 +2,20 @@
 <!DOCTYPE section PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN" "http://www.oasis-open.org/docbook/xml/4.5/docbookx.dtd" [
  <!ENTITY % BOOK_ENTITIES SYSTEM "Admin_Guide.ent">
 ]>
-<section id="admin.config.soap">
-	<title>SOAP API</title>
+<section id="admin.config.api">
+	<title>API</title>
 
-	<para>MantisBT exposes a SOAP API which allows remote clients to interact with MantisBT and perform many of the usual tasks, such as reporting issues, running filtered searches and retrieving attachments. The SOAP API is enabled by default and available at <literal>/api/soap/mantisconnect.php</literal> below your installation root.</para>
-	<para>A WSDL file which describes the web service is available at <literal>/api/soap/mantisconnect.php?wsdl</literal> below your installation root.</para>
+	<para>MantisBT exposes a webservice API which allows remote clients to interact with MantisBT and perform many of the usual tasks, such as reporting issues, running filtered searches and retrieving attachments.</para>
+	<para>
+		The SOAP API is enabled by default and available at <literal>/api/soap/mantisconnect.php</literal>
+		below the MantisBT root.  A WSDL file which describes the web service is available at
+		<literal>/api/soap/mantisconnect.php?wsdl</literal> below the MantisBT root.
+	</para>
+	<para>
+		The REST API is experimental and is disabled by default.
+		A Swagger sandbox and documentation for REST API is available at <literal>/api/rest/swagger/</literal> below
```

Comment:
- `/api/rest/swagger` points to Swagger UI for API documentation and sandbox for making calls to the API from the browser, which helps with understanding the API.
- '/api/rest' is the root end point for the REST API.  The route group (e.g. issues) is added after that.